src/util/apparmor.c

```diff
@@ -10,11 +10,23 @@
  * applicable policy.
  */
 
+#include <c-rbtree.h>
 #include <c-stdaux.h>
+#include <libaudit.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <sys/apparmor.h>
 #include "util/apparmor.h"
+#include "util/audit.h"
 #include "util/error.h"
+#include "util/ref.h"
+#include "bus/name.h"
+
+struct BusAppArmorRegistry {
+        _Atomic unsigned long n_refs;
+        const char *fallback_context;
+        const char *bustype;
```

Comment:
I think both should just drop `const`. Definitely for `bustype`, and I think even for `fallback_context` it would be better to drop it.